### ISSUE TITLE
Mouseless implementation

### ISSUE THREAD

**Issue by julenComposer** [musescore/MuseScore#20281]
### Your idea

Gadually opening Musescore to _fully_ work via keyboard input.

### Problem to be solved

Dear all, I really suggest to consider gradually allowing the user to go full mouseless in Musescore. This would include full navigation between all the items displayed on the score - maybe doable by putting them on different layers -, between all the menus, but most importantly to allow customisable keyboard shortcuts to any element in the palette - with shortcut-setup also fully accessible via keyboard on the fly.

### Additional context

I am a composer, and I used Musescore as my main (read _only_) engraving software for many years now, even for almost unthinkable graphical tasks. It's a very powerful tool, and I would stress how really worth it would be to implement solutions for reducing work friction. And I think full keyboard-only access would be the best one to do.

**Comment by MarcSabatella** [comment]
Have you checked out the Accessibility page in the Handbook?  MuseScore actually *does* provide full keyboard control; it's one of the only music programs that can say that.  All score items, menus, dialogs, etc are all operable by keyboard, and that includes the palette.  The Handbook explains how, and there are tutorial videos as well.

That said, indeed, the ability to assign shortcuts for individual palette items that don't already allow it (a dozen or so of the most common ones already do) is on the list of things we'd all like someday.  See for example https://github.com/musescore/MuseScore/issues/15290

If after consulting the Handbook and watching the tutorials, you find a program with a specific score element or control in the UI that you cannot access by keyboard, please open a separate issue for that particular case.

**Comment by cbjeukendrup** [comment]
I will close this issue for now, since the suggestion about palette item keyboard shortcuts already has its own open issue and as Marc says, in general MuseScore is already (almost) completely keyboard accessible. If you still find specific places that can really not be accessed, please open separate bug reports for that. Thanks!
